code/_onclick/hud/fullscreen.dm

```diff
@@ -7,15 +7,28 @@
 		screen = screens[category]
 		if(screen.type != type)
 			clear_fullscreen(category, FALSE)
-			return .()
+			return
+		else if(screen.clear_after_length)
+			screen = getFromPool(type)
 		else if(!severity || severity == screen.severity)
 			return null
 	else
 		screen = getFromPool(type)
 
-	screen.icon_state = "[initial(screen.icon_state)][severity]"
 	screen.severity = severity
-
+	if(screen.anim_state)
```

Comment:
I'm not entirely sure why this was added
I mean I know WHY because the var itself is pretty descriptive but it worked fine without this before